docs/openstack/backend_services_deployment.md

```diff
@@ -197,11 +197,23 @@ podified OpenStack control plane services.
     ovn:
       enabled: false
       template:
-        ovnController:
-          external-ids:
-            system-id: "random"
-            ovn-bridge: "br-int"
-            ovn-encap-type: "geneve"
+          ovnDBCluster:
+            ovndbcluster-nb:
+              dbType: NB
+              storageRequest: 10G
+              networkAttachment: internalapi
+            ovndbcluster-sb:
+              dbType: SB
+              storageRequest: 10G
+              networkAttachment: internalapi
+          ovnNorthd:
+            networkAttachment: internalapi
+          ovnController:
+            external-ids:
+              system-id: "random"
+              ovn-bridge: "br-int"
+              ovn-encap-type: "geneve"
+            networkAttachment: tenant
```

Comment:
see for the context https://github.com/openstack-k8s-operators/data-plane-adoption/pull/192#issuecomment-1825336591